README.md

```diff
@@ -1,10 +1,24 @@
 # AI studio
 
+## Installing a released version
+
+You can install this extension using this [link](podman-desktop:extension/redhat.ai-studio).
+
+This will install the latest released version of the extension.
+
+If you want to install a specific version of the extension, go to Podman Desktop UI > ⚙ Settings > Extensions > Install a new extension from OCI Image.
+
+The name of the image to use is `ghcr.io/projectatomic/ai-studio:version_to_use`.
+
+You can get released tags for the image at https://github.com/projectatomic/studio-extension/pkgs/container/ai-studio.
```

Comment:
Is this URL correct or you want to link to the release/tags page where they can look the version?

Reply:
If not, can we just have this link in the section below? Why duplicate it?

Reply:
> Is this URL correct or you want to link to the release/tags page where they can look the version?

Image tags and release tags are different: the release tags have a v in front

Reply:
> If not, can we just have this link in the section below or here? Why duplicate it?

Because you can't install a nightly build using podman-desktop: scheme